### DISCUSSION TITLE
ast.Node to cue.Value

### DISCUSSION THREAD

**Discussion by dcb1001** [cue-lang/cue#3122] [Q&A] [answered]
Is there a method to get cue.Value from ast.Node?

**Comment by mvdan** [comment, marked as answer]
You want either https://pkg.go.dev/cuelang.org/go@v0.8.2/cue#Context.BuildExpr or https://pkg.go.dev/cuelang.org/go@v0.8.2/cue#Context.BuildFile. Those mirror the types returned from the parsing APIs at https://pkg.go.dev/cuelang.org/go/cue/parser.

**Reply by dcb1001** [reply]
Thanks !
